pandas/_libs/khash.pxd

```diff
@@ -53,9 +53,9 @@ cdef extern from "khash_python.h":
     kh_pymap_t* kh_init_pymap()
     void kh_destroy_pymap(kh_pymap_t*)
     void kh_clear_pymap(kh_pymap_t*)
-    khuint_t kh_get_pymap(kh_pymap_t*, PyObject*)
+    khuint_t kh_get_pymap(kh_pymap_t*, object)
```

Comment:
Is this just a cleanup or did it actually change the behavior in some way? I have a slight preference for the extern declarations to match exactly the way they are declared in the source headers to minimize having to interpret what is happening; at the end of the day the linker isn't going to type check just resolve the symbols